lua/neogit/status.lua

```diff
@@ -937,6 +980,63 @@ local cmd_func_map = function()
         vim.cmd("split " .. item.name)
       end
     end,
+    ["GoToPreviousHunkHeader"] = function()
```

Comment:
Ok, I *really* like this feature ❤️

Reply:
:D Yeah! It's _slightly_ wonky when you're at the bottom/top of a file, but thats just an issue of fine-tuning